portfolio/src/main/webapp/style.css

```diff
@@ -125,62 +120,74 @@ div.expText {
 }
 
 div.expInformatic img {
-    max-width: 80%;
     margin-left: 10%;
-    padding: 10px;
     margin-right: auto;
+    max-width: 80%;
+    padding: 10px;
 }
 
 
 
 /* gallery stylers */
 .row {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 0 4px;
-  margin-left: 20%;
-  margin-right: 20%;
-  margin-top: 20px;
+    display: flex;
+    flex-wrap: wrap;
+    margin-left: 20%;
+    margin-right: 20%;
+    margin-top: 20px;
+    padding: 0 4px;
 }
 
 .column {
-  flex: 50%;
-  padding: 4px;
+    flex: 50%;
+    padding: 4px;
 }
 
 .column img {
-  margin-top: 8px;
-  vertical-align: middle;
-  max-width: 33%;
+    margin-top: 8px;
+    max-width: 33%;
+    vertical-align: middle;
 }
 
 /* Hobbie and project */
 div.content {
-    margin: 20px;
     border-style: solid;
+    margin: 20px;
     padding: 10px;
     text-align: center;
 }
 
-/**/
-
-
-
 div.captionedImage {
-    width: 100%;
-    margin-left: auto;
-    margin-right:auto;
     background-color: black;
     color: white;
+    margin-left: auto;
+    margin-right:auto;
+    width: 100%;
 }
 
 div.captionedImage img {
     max-width: 100%;
 }
 
 div.caption {
-    text-align: center;
     padding: 10px 20px;
+    text-align: center;
+}
+
+/* comments styles */
+
+#commentsContainer {
+    margin-top: 120px;
+    width: 50%;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
please alphabetize :)
and use shorthand properties when possible (https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)